.github/workflows/continuous-integration.yml

```diff
@@ -466,26 +482,24 @@ jobs:
         env:
           YARN_CACHE_FOLDER: .cache/yarn
 
-      - name: Run git diff and set DIFF_RESULTS
-        if: ${{ github.ref != 'refs/heads/master' }}
-        id: git-diff
-        run: echo ::set-output name=DIFF_RESULTS::$(git diff ${{ github.event.before }} ${{ github.event.after }} --line-prefix=new_line --unified=0)
-
-      - name: Set IS_GRAPH_UPDATED and update the cross-app import graph if necessary
+      - name: Regenerate the cross-app import graph
         if: ${{ github.ref != 'refs/heads/master' }}
         run: node script/github-actions/create-cross-app-import-graph.js
-        env:
-          DIFF_RESULTS: ${{ steps.git-diff.outputs.DIFF_RESULTS }}
 
       - name: Push the updated cross-app import graph
```

Comment:
Should we push the graph only when there's a change? Won't this create too much noise in the git log if it's being committed on each push? This also complicates the git workflow because the branch owner will always have to pull down the graph changes before pushing a new commit.

Reply:
That's what happens - it's only pushed if there's a change. When the graph is regenerated and there's no change, there's nothing to commit. It's like doing `get add .` when nothing has been changed. There's nothing to stage.

The branch owner will only have to `git pull --rebase` if the graph changes. I'm guessing the graph isn't going to change all that often, but I'm not sure.